### PR TITLE
Add errors.Wrap to indicate the file relabel failed on

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@mrunalp @crosbymichael @vrothberg @giuseppe PTAL

File: go-selinux/selinux_linux.go [hunk 1]
```diff
@@ -289,7 +289,10 @@ func SetFileLabel(fpath string, label string) error {
 	if fpath == "" {
 		return ErrEmptyPath
 	}
-	return lsetxattr(fpath, xattrNameSelinux, []byte(label), 0)
+	if err := lsetxattr(fpath, xattrNameSelinux, []byte(label), 0); err != nil {
+		return errors.Wrap(err, "Failed to set file label on %s")
```
Comment (giuseppe) [inline go-selinux/selinux_linux.go]: It is missing the argument `fpath`.

Should the error start with a lowercase letter?

Comment (vrothberg) [reply go-selinux/selinux_linux.go]: > Should the error start with a lowercase letter?

By convention yes. Lowercase makes it easier to read when being wrapped. Only logs should start with upper case.

Comment (rhatdan) [reply go-selinux/selinux_linux.go]: Fixed

Comment (giuseppe) [reply go-selinux/selinux_linux.go]: don't see a new version yet, was it pushed?

Comment (rhatdan) [reply go-selinux/selinux_linux.go]: It is there now.

**Comment by crosbymichael** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**